multicast/__init__.py

```diff
@@ -590,6 +590,7 @@ def buildArgs(cls, calling_parser_group: argparse.ArgumentParser) -> argparse.Ar
 			)
 			calling_parser_group.add_argument(
 				"--use-std", dest="is_std", default=False, action="store_true",
+				help="Use interactive command mode. Disabled by default."
```

Comment:
> [!NOTE]
> This change is a clarification to help distinguish between `--pipe` and `--use-std` which both impact interactions with standard i/o on the command line.

@coderabbitai FYI, I've opened GHI #393 to followup on the mentioned 'interactive mode' in the future.